pkg/eks/api.go

```diff
@@ -257,6 +257,14 @@ func (c *ClusterProvider) EnsureAMI(version string, ng *api.NodeGroup) error {
 		return ami.NewErrNotFound(ng.AMI)
 	}
 
+	// Get Root Device Name
+	rootDevice, err := ami.LookupRootDeviceName(c.Provider.EC2(), ng.AMI)
+	if err != nil {
+		return errors.Wrapf(err, "%s failed root device name lookup", ng.AMI)
```

Comment:
I think that since `ami.LookupRootDeviceName()` returns a specific error message here we could just return that error without an extra message. 

@errordeveloper what do you think?

Reply:
I'd say `unable to get root device name for image %q`.

Reply:
This has been merged into IsAvailable.